examples/amm/src/contract.rs

```diff
@@ -583,7 +583,7 @@ impl AmmContract {
 
             // When removing liquidity, you'll specify one of the tokens you want to
             // remove and the amount, and we'll calculate the amount for the other token that
-            // we'll remove based on the current ratio, and remove them.
+            // we'll remove them based on the current ratio, and remove them.
```

Comment:
That one I'm not sure about:
"we'll calculate the amount … that we'll remove"
I.e. we'll remove the _amount_, so no "them" is needed?